sql/catalyst/src/main/scala/org/apache/spark/sql/types/LongType.scala

```diff
@@ -43,7 +43,7 @@ class LongType private() extends IntegralType {
    */
   override def defaultSize: Int = 8
 
-  override def simpleString: String = "bigint"
+  override def simpleString: String = "long"
```

Comment:
When you try to read a csv and map to a case class with a Long your get a message like this one:
__EXCEPTION__:org.apache.spark.sql.AnalysisException: Cannot up cast linked_docs.`MR_NUMBER_OF_DOCS_UPLOADED` from string to bigint as it may truncate
The type path of the target object is:
- field (class: "scala.Long", name: "MR_NUMBER_OF_DOCS_UPLOADED")

Getting a message that talks about bigint while you are trying to cast a String to a Long looks confusing to me. I thought this was a typo.